third_party/BUILD

```diff
@@ -598,6 +598,17 @@ filegroup(
     ],
 )
 
+filegroup(
+    name = "java_tools",
+    srcs = glob(
+        ["java/java_tools/*.jar"],
+        exclude = ["java/java_tools/SingleJar_deploy.jar"]
```

Comment:
This change will need to be split (to a Git-on-Borg and a Google internal part). (But it's easier to review in one piece!)

Reply:
Why does it need to be split?